aggregator/src/test/java/com/redhat/cloud/notifications/DailyEventAggregationJobTest.java

```diff
@@ -0,0 +1,400 @@
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectSpy;
+import io.smallrye.reactive.messaging.memory.InMemoryConnector;
+import io.smallrye.reactive.messaging.memory.InMemorySink;
+import jakarta.enterprise.inject.Any;
+import jakarta.inject.Inject;
+import org.eclipse.microprofile.reactive.messaging.Message;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static com.redhat.cloud.notifications.models.SubscriptionType.DAILY;
+import static java.time.ZoneOffset.UTC;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+@QuarkusTest
+@QuarkusTestResource(TestLifecycleManager.class)
+class DailyEventAggregationJobTest {
+
+    @Inject
+    ResourceHelpers helpers;
+
+    @Inject
+    ResourceHelpers resourceHelpers;
+
+    @InjectSpy
+    DailyEmailAggregationJob dailyEmailAggregationJob;
+
+    @Inject
+    @Any
+    InMemoryConnector connector;
+
+    @Inject
+    ObjectMapper objectMapper;
+
+    @InjectSpy
+    AggregatorConfig aggregatorConfig;
+
+    LocalDateTime baseReferenceTime;
+
+    final AggregationOrgConfig someOrgIdToProceed = new AggregationOrgConfig("someOrgId",
+            LocalTime.of(LocalTime.now(ZoneOffset.UTC).getHour(), LocalTime.now(ZoneOffset.UTC).getMinute()),
+            LocalDateTime.now(ZoneOffset.UTC).minusDays(1));
+
+    final AggregationOrgConfig anotherOrgIdToProceed = new AggregationOrgConfig("anotherOrgId",
+            LocalTime.of(LocalTime.now(ZoneOffset.UTC).getHour(), LocalTime.now(ZoneOffset.UTC).getMinute()),
+            LocalDateTime.now(ZoneOffset.UTC).minusDays(1));
+
+    @BeforeEach
+    void setUp() {
+        helpers.purgeEventAggregations();
+        initAggregationParameters();
+        when(aggregatorConfig.isAggregationBasedOnEventEnabled()).thenReturn(true);
+        baseReferenceTime = dailyEmailAggregationJob.computeScheduleExecutionTime();
+
+        when(dailyEmailAggregationJob.computeScheduleExecutionTime()).thenReturn(baseReferenceTime);
+    }
+
+    @AfterEach
+    void tearDown() {
+        helpers.purgeEventAggregations();
+        connector.sink(DailyEmailAggregationJob.EGRESS_CHANNEL).clear();
+    }
+
+    void initAggregationParameters() {
+        helpers.purgeAggregationOrgConfig();
+        dailyEmailAggregationJob.defaultDailyDigestTime = LocalTime.now(ZoneOffset.UTC);
+    }
+
+    List<AggregationCommand> getRecordsFromKafka() {
```

Comment:
There are multiple occurences in this file where the generic type is missing:
```suggestion
    List<AggregationCommand<EventAggregationCriteria>> getRecordsFromKafka() {
```
I'm not sure it's worth the effort fixing this as it's not runtime code. I'll let you decide if you want to fix it or not!